test/fuzz/main/README.md

```diff
@@ -0,0 +1,658 @@
+# Harry, a fuzz testing tool for Apache Cassandra
+
+The project aims to generate _reproducible_ workloads that are as close to real-life as possible, while being able to
+_efficiently_ verify the cluster state against the model without pausing the workload itself.
+
+## Getting Started in under 5 minutes
+
+Harry can operate as a straightforward read/write "correctness stress tool" that will check to ensure reads on a cluster
+are consistent with what it knows it wrote to the cluster. You have a couple options for this.
+
+### Option 2: Running things manually lower in the stack:
+
+The make file has a stress target where you can more directly access all available ARGS rather than restricting yourself
```

Comment:
We didn't bring over the `Makefile` right?

Reply:
Well spotted; there's no makefile _for now_. I will bring it in eventually. Maybe in a different form.